ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/DbChangelog.hs

```diff
@@ -330,7 +343,9 @@ flush ::
   -> DbChangelog l
   -> (Maybe (DbChangelogToFlush l), DbChangelog l)
 flush policy dblog =
```

Comment:
The comment is probably a bit outdated. We should check if this is the case in other places too